.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+staleLabel: stale
+daysUntilClose: 7
+
+pulls:
+    daysUntilStale: 180 # ~6 months
+    markComment: >
+        This pull request has been automatically marked as stale because it has not had
+        recent activity. If you'd still like this PR merged, please comment on the task,
+        make sure you've addressed reviewer comments, and rebase on the latest master.
+        Thank you for your contributions!
+
+issues:
```

Comment:
not sure what our philosophy is on issues, but I took a stab at a configuration.